tracer/test/Datadog.Trace.Debugger.IntegrationTests/Helpers/DebuggerSampleProcessHelper.cs

```diff
@@ -36,6 +39,13 @@ internal async Task StopSample()
                 throw new InvalidOperationException($"The process did not exit after {timeout}ms");
             }
 
+            if (Process.ExitCode != 0)
+            {
+                _output.WriteLine($"[DebuggerSampleProcessHelper] Process exited with code {Process.ExitCode} (0x{Process.ExitCode:X})");
```

Comment:
I'm not sure if `_output` is _always_ non null? 🤔 If it is this is all good, otherwise we should maybe add a null check in the if block 🙂